specs/core/0_beacon-chain.md

```diff
@@ -587,8 +588,9 @@ def deposit(deposit_params: bytes[2048]):
     for i in range(POW_CONTRACT_MERKLE_TREE_DEPTH):
         index //= 2
         self.receipt_tree[index] = sha3(concat(self.receipt_tree[index * 2], self.receipt_tree[index * 2 + 1]))
-    self.total_deposit_count += 1
-    if self.total_deposit_count == 16384:
+    if msg.value >= MIN_DEPOSIT_SIZE:
```

Comment:
`MIN_DEPOSIT_SIZE` has not been defined. Do you mean `as_wei_value(DEPOSIT_SIZE, "ether")`?
```suggestion
    if msg.value >= as_wei_value(DEPOSIT_SIZE, "ether"):
```